doc/installation.md

```diff
@@ -239,9 +239,14 @@ e.g.
     ```sh
     cd <espnet-root>/tools
     . activate_python.sh
-    . ./setup_cuda_env.sh <cuda-root>  # e.g. <cuda-root> = /usr/local/cuda
     ./installers/install_warp-transducer.sh
```

Comment:
This line is required

Reply:
Hum, I can install both `warp-transducer` and `warp-rnnt` without executing `setup_cuda_env.sh`?

Reply:
It's required.